pyproject.toml

```diff
@@ -27,6 +27,7 @@ cryptography = ">=3.4.8"
 tqdm = "^4.32.1"
 pydantic = ">=1.10.14"
 orjson = ">=2"
+ruff = "*"
```

Comment:
I don't think this is needed, especially as non-dev dependency. Let's remove it.
```suggestion
```